tests/proc/test_manager.py

```diff
@@ -74,7 +79,7 @@ def test_kill(
         )
 
     mock_kill.reset_mock()
-    process_manager.kill(finished_process)
+    assert process_manager.kill(finished_process) is False
```

Comment:
tests for failed should just be
```
with pytest.raises(ProcessLookupError)
```
after the other suggested changes

(and we won't have to test `is True` any more)